src/D_EcobeeHouse1.json

```diff
@@ -35,10 +35,10 @@
     ],
     "inScene": "1",
     "DisplayStatus": {
-        "Service": "urn:upnp-org:serviceId:SwitchPower1",
-        "Variable": "Status",
-        "MinValue": "0",
-        "MaxValue": "1"
+        "Service": "urn:ecobee-com:serviceId:Ecobee1",
+        "Variable": "currentClimateRef",
+        "MinValue": "home",
+        "MaxValue": "smart1"
```

Comment:
An alternative approach that solves the issue would be to provide users a device variable on the thermostat device whose default value is `smart1` but the user could change it through advanced settings to the correct climate ref that holds the meaning of "vacation".  Also curious: why did you not use the built-in vacation concept exposed by the API instead of a climate/comfort setting?  (As I'm not a user of the plugin these days, I defer to you and the active user community on this subject!)